src/main/resources/story/web/CreateAccount.story

```diff
@@ -0,0 +1,38 @@
+Description: Sign into www.imdb.com registered account, add movie to whachlist and export it
+
+Scenario: Sign into  https://www.imdb.com/account, add movie to watchlist and export it from watchlist.
+Meta:
+    @layaout desctop
+    @skip
+
+
+Given I am on the main application page
+When I click on element located `By.xpath(//div[contains(text(),'Sign In')])`
+When I click on element located `By.xpath(//span[contains(text(),'Sign in with IMDb')])`
+When I enter `${TestEmail}` in field located `By.xpath(//input[@id='ap_email'])`
+When I enter `${TestPassword}` in field located `By.xpath(//input[@id='ap_password'])`
+When I click on element located `By.xpath(//input[@id='signInSubmit'])`
+When I click on element located `By.xpath(//input[@id='suggestion-search'])`
+When I enter `The Boss Baby` in field located `By.xpath(//input[@id='suggestion-search'])`
+When I click on element located `By.xpath(//button[@id='suggestion-search-button'])`
+When I click on element located `By.xpath(//body[1]/div[3]/div[1]/div[2]/div[3]/div[1]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[2]/a[1])`
```

Comment:
Xpathes should be sensible - by such an xpath it's not clear what you check. Try to shorten it and add meaningfull part.